manual/src/html_processing/scss/style.scss

```diff
@@ -793,8 +793,7 @@
 
     code span.constructor,
     .caml-input span.kw2 {
-	font-weight: 500;
-	color: #a28867;
+	color: #8d543c;
```

Comment:
I may have had another reason I can't remember, but I don't see why it would be bolded in the first place and it's not bolded in htmlman. I think now only keywords like `let` and `while` are bolded, which I think is standard when displaying code.

It's also annoying that `.constructor` in the HTML applies to both data constructors and modules, having a different style for modules (maybe just bolded) would be nice.